Content.Server/Chemistry/EntitySystems/ChemistrySystem.Injector.cs

```diff
@@ -188,7 +187,7 @@ private void OnInjectorAfterInteract(EntityUid uid, InjectorComponent component,
 
     private void OnInjectorStartup(EntityUid uid, InjectorComponent component, ComponentStartup args)
     {
-        /// ???? why ?????
+        // Necessary for the Client-side InjectorStatusControl to get the appropriate Volume values.
         Dirty(component);
```

Comment:
No, this shouldn't be needed to do that. The fact that this is apparently required just means that there is something else wrong with InjectorStatusControl or something else. An component should never have to just dirty itself on startup

Reply:
This appears to be a recurring pattern.

https://github.com/space-wizards/space-station-14/blob/f1ab5c891cc47658c8711e8b7b44a216cccbe6b5/Content.Server/Tools/ToolSystem.Welder.cs#L166-L169

https://github.com/space-wizards/space-station-14/blob/f1ab5c891cc47658c8711e8b7b44a216cccbe6b5/Content.Server/Crayon/CrayonSystem.cs#L124-L132

https://github.com/space-wizards/space-station-14/blob/f1ab5c891cc47658c8711e8b7b44a216cccbe6b5/Content.Server/DeviceNetwork/Systems/DeviceListSystem.cs#L20-L23

https://github.com/space-wizards/space-station-14/blob/f1ab5c891cc47658c8711e8b7b44a216cccbe6b5/Content.Shared/Light/SharedHandheldLightSystem.cs#L27-L33

https://github.com/space-wizards/space-station-14/blob/f1ab5c891cc47658c8711e8b7b44a216cccbe6b5/Content.Shared/Weapons/Ranged/Systems/SharedGunSystem.BasicEntity.cs#L33-L42

https://github.com/space-wizards/space-station-14/blob/f1ab5c891cc47658c8711e8b7b44a216cccbe6b5/Content.Server/Research/Systems/ResearchSystem.Server.cs#L16-L22